cpp/package/package.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+#
+#  Package nebula as deb/rpm package
+#
+# introduce the args
+#   -v: The version of package, the version should be match tag name, default value is the `commitId`
+#   -n: Package to one or multi packages, `ON` means one package, `OFF` means multi packages, default value is `ON`
+#   -s: Whether to strip the package, default value is `FALSE`
+#   -a: Whether to disable the cxx11 abi, default value is `OFF`
+#
+# usage: ./package.sh -v <version> -n <ON/OFF> -s <TRUE/FALSE> -b <BRANCH>
```

Comment:
here is no need `-n`, the enable to package to one or multi files

Reply:
Don't plan to modify the script too much.